SpatialGDK/Extras/schema/spatial_debugging.schema

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+component SpatialDebugging {
```

Comment:
Is it sufficient for us to just send the colours rather than the virtual worker IDs also?

Reply:
Given that the colors can sometimes be really similar, I was thinking we might want to have the box be a color and then the virtual worker ID as text inside the colored box.

Reply:
Sounds good, raised https://improbableio.atlassian.net/browse/UNR-2774